constants/Header/constants.js

```diff
@@ -0,0 +1,40 @@
+import ICONS from "../../components/icons/icons";
+import ROUTES from "../../routes/routes";
+import headerTheme from "../../themes/Header/theme";
+
+const CONSTANTS = {
```

Comment:
This is too generic, you can create a routes file for defining all your routes

Reply:
Done

Reply:
Done